qa/tasks/cephfs/test_volumes.py

```diff
@@ -6330,3 +6330,143 @@ def test_subvolume_upgrade_v1_to_v2(self):
 
         # verify trash dir is clean
         self._wait_for_trash_empty()
+
+    def test_malicious_metafile_on_legacy_to_v1_upgrade(self):
+        """
+        Validate handcrafted .meta file on legacy subvol root doesn't break the system
+        on legacy subvol upgrade to v1
+        poor man's upgrade test -- theme continues...
+        """
+        subvol1, subvol2 = self._generate_random_subvolume_name(2)
+        group = self._generate_random_group_name()
```

Comment:
@kotreshhr This variable is assigned but not used, which breaks `run-tox-qa` test.

Reply:
@idryomov my bad. @djgalloway would you fix this ? I can do that. Do I have permission to push to your branch ?

Reply:
Yes, of course.

Reply:
@kotreshhr you have permission to push to this branch.  I don't know how to fix the test.

Reply:
Are you suggesting just cherry-picking https://github.com/ceph/ceph/pull/47229/commits/c8416c3f0e15e49dde4b9bca6f732231d9b5d16b now?

Reply:
Yes, but preferably after https://github.com/ceph/ceph/pull/47229 is merged.